### ISSUE TITLE
msh fails on start

### ISSUE THREAD

**Issue by CheeseBender** [gekware/minecraft-server-hibernation#130]
Nothing else to say. This is the log I receive when trying to execute the code.

```
╔═════════════════════════════════════════════╗
║  _ __ ___  ___| |__                         ║
║ | '_ ` _ \/ __| '_ \                        ║
║ | | | | | \__ \ | | |                       ║
║ |_| |_| |_|___/_| |_| v2.4.5                ║
║ Copyright (C) 2019-2022 gekigek99           ║
║ github: https://github.com/gekigek99        ║
║ remember to give a star to this repository! ║
╚═════════════════════════════════════════════╝
2022/02/07 02:57:47 [info  *   ] checking OS support...
2022/02/07 02:57:47 [info  *   ] loading config default...
2022/02/07 02:57:47 [info  *   ] loading config runtime...
2022/02/07 02:57:47 [info  ****] log level set to: 1
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x24f0a0]

goroutine 8 [running]:
msh/lib/progmgr.checkUpdate(0x1, 0x305a94, 0x6, 0x3087d8, 0x10, 0x0, 0x0, 0x0, 0x0)
        /home/opc/minecraft-server-hibernation/lib/progmgr/progmgr.go:142 +0x488
msh/lib/progmgr.UpdateManager(0x305a94, 0x6)
        /home/opc/minecraft-server-hibernation/lib/progmgr/progmgr.go:79 +0x8c
created by main.main
        /home/opc/minecraft-server-hibernation/minecraft-server-hibernation.go:45 +0xfc
```

**Comment by gekigek99** [comment]
The issue is being investigated.

Did you compile yourself the executable? At which commit?

**Comment by gekigek99** [comment]
the bug has been fixed in 18701ed67970b4b106a32fe3d61bb67402ce84c8

thanks a lot for reporting
